libs/platform/ebpf_hash_table.c

```diff
@@ -224,8 +226,17 @@ _ebpf_hash_table_replace_bucket(
             goto Done;
         }
         delete_data = new_data;
-        if (data) {
-            memcpy(new_data, data, hash_table->value_size);
+        if (value) {
```

Comment:
If you plan on moving the code out of here in a later commit, please add an issue marker.

Reply:
Done